src/Components/Scales/Popover/scale-events.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Popover } from "@mui/material";
+// import IconEye from '../../Assets/icon_eye_scale.svg'
+// import IconCalendar from '../../Assets/icon_white_calendar.svg'
+
+import { IconEye } from "./Icons/icon-eye";
+import { IconCalendar } from "./Icons/icon-calendar";
+import { IconTrash } from "./Icons/icon-trash";
+
+interface IScaleEventsProps {
+    id: string | undefined,
+    open: boolean,
+    anchorEl: HTMLButtonElement | null,
+    setAnchorEl: React.Dispatch<React.SetStateAction<HTMLButtonElement | null>>
+}
+
+const styleButtons = {
+    display: 'flex',
+    justifyContent: 'start',
+    width: '250px',
+    maxHeight: '50px',
+    fontSize: '1rem',
+    fontWeight: '600',
+    textTransform: 'none',
+    borderRadius: '12px',
+    boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.1)',
+}
+
+export const ScaleEvents = (props: IScaleEventsProps) => {
+    const { id, anchorEl, open, setAnchorEl } = props
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    return (
+        <Popover
+            id={id}
+            open={open}
+            anchorEl={anchorEl}
+            onClose={handleClose}
+            anchorOrigin={{
+                vertical: 'center',
+                horizontal: 'left',
+            }}
+        >
+            <div style={{
+                display: "flex",
+                flexDirection: "column",
+                alignItems: 'center',
+                margin: "20px 20px 20px 20px",
+            }}>
+                <Button
```

Comment:
Quanto ao elemento `<Button />`, vou deixar alguns pontos de melhoria: 
1. Há duas propriedades de estilização, o `style={}` e `sx={}`. Dá pra reduzir esse código utilizando o operador spread do javascript, pesquisa sobre: "How to use spread with sx property material UI", isso vai te mostrar como fazer isso com menos linhas. 
2. Outro ponto que deixo pra te pensar como resolver: o objeto passado dentro do style em três buttons se repete, também dá reduzir nesse ponto, veja como fazer isso.
3. Remover o código comentado de todo o arquivo, como o `onClick` que está comentados nos buttons, assim como as importações que não estão em uso.